### PR TITLE
Update documentation for CreateLogGroup support in awslogs

### PR DESCRIPTION
**Pull request by yongtang**
This fix updates the documetation for CreateLogGroup support in awslogs

This fix is related to
https://github.com/docker/docker/issues/29334
https://github.com/docker/docker/pull/29504

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

/cc @thaJeztah

### CODE REVIEW

File: engine/admin/logging/awslogs.md [hunk 1]
```diff
@@ -58,6 +58,17 @@ specified, the container ID is used as the log stream.
 > at a time.  Using the same log stream for multiple containers concurrently
 > can cause reduced logging performance.
 
+### awslogs-create-group
+
+Log driver will return an error by default if the log group does not exist. However, if you want to
```
Comment (mstanleyjones) [inline engine/admin/logging/awslogs.md]: Please reword this to be a bit more concise. Something like:

However, you can set the `awslogs-create-group` to `true` to automatically create the log group as needed. The `awslogs-create-group` option defaults to `false`.

Note: Your AWS IAM policy must include the `logs:CreateLogGroup` permission before you attempt to use `awslogs-create-group`.

**Comment by yongtang** [comment]
Thanks @mstanleyjones. The PR has been updated. Please take a look.
